### ISSUE TITLE
After "Insert frame", frames go out of sync with values of Function/Transparency

### ISSUE THREAD

**Issue by d01010101** [opentoonz/opentoonz#3486]
"Insert a frame" (either in the frame editor or in the function editor) does not shift down values in "Transparency/Intensity".

1. New scene.
2. Add a level and some frames, not necessary.
3. FxSchematic -> create an FX "Transparence" anywhere.
4. Function editor -> enable the column "Intensity", enter a value anywhere.
5. At a frame before the value of intensity, right button -> "Insert frame"
6. A new frame is created in the frame editor, everything shifts down in that editor as necessary.
7. The intensity value in the function editor rests where it was before.

It makes frames and fx out of sync.

 - OS: Linux or Windows
 - OpenToonz Version: 1.4 and OTX from September (the latter tested only under Linux)

**Comment by DarrenTAnims** [comment]
This is part of an ongoing issue that changing the exposure of drawings on the xsheet doesn't, by default, affect the animation keys shown on the xsheet nor the fx keys. Personally, I would prefer the adjusting of the drawings on the xsheet to directly affect the keys shown in the function editor.

**Comment by d01010101** [comment]
Is there any workaround? I am stuck with my animation. Any, even cumbersome workaround, so that I can cleanly insert 100 frames in front of my animation?

There is an option about moving keys together with frames, it is enabled and works in some cases, but I have found no way of moving fx transparency values when a frame is inserted. Can I move these, at least manually for now?

**Comment by DarrenTAnims** [comment]
It's not really a workaround, but you can move your animation frames in the function editor with either click and drag or cut/paste

**Comment by d01010101** [comment]
Unless I do something wrong, cut&paste in the function editor moves keys but not frames, so these two go out of sync. As of drag&drop, I do not know how to do this. Trying to drag a selection, or a small tab, in any editor, just makes a new selection, at least in Linux.

**Comment by DarrenTAnims** [comment]
Yes, cut and paste moves keys. That's all it does. It doesn't affect the drawings. You have to move the drawings and keys separately.

You only drag the keys. You have to highlight them, then drag them. It is tricky to do. That control isn't as good as it could be for moving the keys.

**Comment by RodneyBaker** [comment]
@d01010101 Can you share some screen captures or video of what you are referring to.
The imagery will go a long way in conveying the problem.
Thanks!
